droid-hal-device.inc

```diff
@@ -815,6 +815,7 @@ fi
 
 %files -f tmp/droid-hal.files
 %defattr(-,root,root,-)
+/data
```

Comment:
This should be %dir /data to ensure we only include the directory for this package, right?

Reply:
Done